test_extract_unique_sensors_01.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import unittest
+from pathlib import Path
+from urllib.request import urlopen
+
+from mock import MagicMock, Mock, mock_open, patch
+
+from extract_unique_sensors_01 import (extract_all_sensors, read_sensors,
+                                       splitting_extracted_sensors)
+
```

Comment:
I ran [isort](https://github.com/PyCQA/isort) which is a nifty tool for sorting out imports.

Command ran:
```bash
isort -rc .
```